webview_panels/src/modules/defer/DeferToProduction.tsx

```diff
@@ -140,25 +251,75 @@ const DeferToProduction = (): JSX.Element => {
                 />
               </Label>
             </FormGroup>
-            <FormGroup>
-              <Stack>
+            <Label>Save your file location</Label>
```

Comment:
It would have been better to extract this whole manifest selection logic to a separate component to avoid bloating this defer component